Restyle

omercnet opened the pull request.

barnhartguy commented:
1. I think we should use round corner for the buttons, it looks more intuitive
2. in the register page the buttons are touching each other
3. the agenda page looks bland, i suggest using zebra table. also it is corrupted on mobile
4. please update the slack and register buttons in the CTF to the same formatting
5. there was a request to make the buttons larger by the team, i assume the designer thought it looked bad - if not, please consider restoring the larger size

K3r3n3 commented:
Restyle website comments 

1. top menu is hard to read (see photo), need more contrast between text of menu and the background
2. front page - background photo behind the logo is too dark to see clearly 
3. front page - logo needs to be resized - top of the logo (mask) is cut by the menu
4. agenda page - hard to read the time, maybe use a table or smaller font so it doesn't spill into  2 rows
5. front page + register page : I think the slack + register buttons look a bit "stuck on" , disjointed from the rest of the site. Maybe rounded corners will help? or different background?  I dont know. 
6. speaker page + team page randomize the list of speakers & team each time I reload it . Intentional? 
7. ctf page - CTF banner needs to be resized, spills outside of the screen, see screenshot

![image](https://user-images.githubusercontent.com/67376368/85579886-edfbfd00-b643-11ea-8543-c7ed91e1c838.png)

![image](https://user-images.githubusercontent.com/67376368/85581222-11737780-b645-11ea-964e-e475d0a7ae3d.png)

omercnet commented:
fixed buttons according to design guidelines, updated ctf
menu items are all bright, not just selected one
background photo is now 10% more visible

@K3r3n3 
> front page - logo needs to be resized - top of the logo (mask) is cut by the menu
this only happens on vertically challenges laptops, if you zoom out it'll be fine
complicated to fix, will try again later

> speaker page + team page randomize the list of speakers & team each time I reload it . Intentional?
yes

CTF pages will be worked on separately
